clipper_admin/clipper_admin/kubernetes/kubernetes_container_manager.py

```diff
@@ -646,6 +665,23 @@ def stop_all_model_containers(self):
                     label=CLIPPER_MODEL_CONTAINER_LABEL,
                     cluster_label=CLIPPER_DOCKER_LABEL,
                     cluster_name=self.cluster_name))
+
+            self._k8s_beta.delete_collection_namespaced_replica_set(
+                namespace=self.k8s_namespace,
+                label_selector="{label}, {cluster_label}={cluster_name}".
```

Comment:
It seems there are redundant codes. Would like to define `label_selector` as a variable?

Reply:
Done